modules/ipi-install-bmc-addressing.adoc

```diff
@@ -64,3 +64,62 @@ platform:
           password: <password>
           disableCertificateVerification: True
 ----
+[discrete]
+== Redfish APIs
+
+Several redfish API endpoints are called onto your BCM when using the bare-metal installer-provisioned infastructure.
+
+[IMPORTANT]
+====
+You need to ensure that your BMC supports all of the redfish APIs prior to installation.
+====
+
+**List of redfish APIs:**
+
+.PowerOn
+[source, terminal]
+----
+curl -u $USER:$PASS -X POST -H'Content-Type: application/json' -H'Accept: application/json' -d '{"Action": "Reset", "ResetType": "On"}' https://$SERVER/redfish/v1/Systems/$SystemID/Actions/ComputerSystem.Reset
+----
+
+.PowerOff
+[source, terminal]
+----
+curl -u $USER:$PASS -X POST -H'Content-Type: application/json' -H'Accept: application/json' -d '{"Action": "Reset", "ResetType": "ForceOff"}' https://$SERVER/redfish/v1/Systems/$SystemID/Actions/ComputerSystem.Reset
+----
+
+.Temporary boot
+[source, terminal]
+----
+curl -u $USER:$PASS -X PATCH -H "Content-Type: application/json"  https://$Server/redfish/v1/Systems/$SystemID/ -d '{"Boot": {"BootSourceOverrideTarget": "pxe", "BootSourceOverrideEnabled": "Once"}}
+----
+
+.BootSourceOverrideMode
+[source, terminal]
+----
+curl -u $USER:$PASS -X PATCH -H "Content-Type: application/json"  https://$Server/redfish/v1/Systems/$SystemID/ -d '{"Boot": {"BootSourceOverrideMode":"UEFI"}}
+----
+
+**List of redfish-virtualmedia APIs:**
+
+[NOTE]
+====
+The `PowerOn` and `PowerOff` commands for redfish APIs are the same for the redfish-virtualmedia APIs.
+====
+
+.Set temporary boot device
+[source, terminal]
+----
+curl -u $USER:$PASS -X PATCH -H "Content-Type: application/json" https://$Server/redfish/v1/Systems/$SystemID/ -d '{"Boot": {"BootSourceOverrideTarget": "cd", "BootSourceOverrideEnabled": "Once"}}'
+----
+
+.Mount virtualmedia
+[source, terminal]
+----
+curl -u $USER:$PASS -X PATCH -H "Content-Type: application/json" -H "If-Match: *" https://$Server/redfish/v1/Managers/$ManagerID/VirtualMedia/$VmediaId -d '{"Image": "https://example.com/test.iso", "TransferProtocolType": "HTTPS", "UserName": "", "Password":""}'
+----
+
+[NOTE]
+====
+We only support TransferProtocolTypes of HTTPS or HTTP.
```

Comment:
Avoid "we" language. Should `TransferProtocolTypes` be in backticks and have a noun following it?

Reply:
Whoop, I put the object and noun together 🙃. 
Thank you for the feedback!!